app/livechat/server/lib/routing/External.js

```diff
@@ -0,0 +1,55 @@
+import { Meteor } from 'meteor/meteor';
+import { HTTP } from 'meteor/http';
+
+import { settings } from '../../../../settings';
+import { RoutingManager } from '../RoutingManager';
+import { Users } from '../../../../models';
+
+class ExternalQueue {
+	constructor() {
+		this.config = {
+			previewRoom: false,
+			showConnecting: false,
+			showQueue: false,
+			returnQueue: true,
```

Comment:
"return to queue" was initially developed so the chat would be sent back to the pool.. so I guess this value should be `false` for `External Queue` (as it was until now)